public/images/jetpack/jetpack-manage.svg

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="149px" height="152px" viewBox="0 0 149 152" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.4 (15588) - http://www.bohemiancoding.com/sketch -->
```

Comment:
I have no idea but could we strip some of the comments from here. Not sure how vital they are to the SVG working correctly. I think the title and the description could go as well. cc @alternatekev or @MichaelArestad